ui/nuxeo-layout.js

```diff
@@ -93,6 +93,9 @@ import './nuxeo-error.js';
       for (let i = 0; i < this.element.__templateInfo.nodeInfoList.length; i++) {
         const nodeInfo = this.element.__templateInfo.nodeInfoList[i];
         const node = this.element.__templateInfo.nodeList[i];
+        if (typeof node.hasAttribute === 'undefined') {
+          break;
```

Comment:
Shouldn't it be `continue;` instead? Otherwise, this won't check any other binding after this one.